internal/updater/online.go

```diff
@@ -18,11 +15,11 @@ import (
 	distlock "github.com/quay/claircore/pkg/distlock/postgres"
 )
 
-// Executor is a helper for managing locks and state for updaters.
+// Online is a helper for managing locks and state for updaters.
```

Comment:
Can we make this "Online is a controller" like the Offline controller comments say?